### PR TITLE
Makes it harder to meta heretics through poppies and kitchen knives

### PR DESCRIPTION
**Pull request by VictorVow**
## About The Pull Request
Poppies are now common maint loot instead of only being found in the Chapel and Botany. 

Poppies can now be crafted with one metal rod to make a poppy pin, an accessory which can be worn to give a very small mood buff in remembrance of those fallen in the Eighth Corporate War. They can also be crafted back to a normal poppy.

Shanks are now a subtype of kitchen knife and thus can be used in the ritual to make a sickly blade. 

## Why It's Good For The Game

Poppies are no longer a dead giveaway of a heretic because they could have been randomly picked up from anywhere and also have a universal benefit, albeit tiny in effect. 

Shanks were a snowflake item instead of more sensibly being a subtype of kitchen knife, and additionally this can be crafted from common materials instead of people noticing you randomly printing a kitchen knife from an autolathe or finding it when searched.

## Changelog
:cl:
tweak: Poppies now spawn in maint loot.
add: Poppy pins can be crafted and give a very small mood buff when worn. 
tweak: Shanks are now a subtype of knife, allowing it to be used by heretics to create their sickly blade. 
/:cl:

### CODE REVIEW

**Comment by francinum** [comment]
These are nice, but I question why we weren't just throwing out players for validing off such unreliable information?

**Comment by heepox** [comment]
> These are nice, but I question why we weren't just throwing out players for validing off such unreliable information?

being seen with poppy / taking poppies makes them very obvious. resulting in search. resulting in validation

**Comment by PowerfulBacon** [comment]
I take poppies all the time and wear them on head but don't get valid hunted.
Also why the shank being a subtype of knife so it works on the runes? Shank is just a piece of glass not a knife specifically

**Comment by VictorVow** [comment]
I don't think poppies have a sprite when worn like that, and from experience poppies are never seen worn/carried outside of heretics.

The ritual hasn't got a prerequisite for metal, since other materials like bone daggers work with it. Probably leaning on my Pale magic system reading lately but I see the heretic rituals as requiring items which are conceptual of something eg it needs a blade, no matter the material and transmutes it. It additionally requires a token which is conceptual of Ash/Flesh/Rust like a match/etc, and transmutes the blade into an Ash/Flesh/Rust blade.

**Comment by PowerfulBacon** [comment]
> 
> 
> I don't think poppies have a sprite when worn like that, and from experience poppies are never seen worn/carried outside of heretics.
> 
> The ritual hasn't got a prerequisite for metal, since other materials like bone daggers work with it. Probably leaning on my Pale magic system reading lately but I see the heretic rituals as requiring items which are conceptual of something eg it needs a blade, no matter the material and transmutes it. It additionally requires a token which is conceptual of Ash/Flesh/Rust like a match/etc, and transmutes the blade into an Ash/Flesh/Rust blade.

they do have a head sprite

**Comment by Sapfearon** [comment]
You just made sentient elite reward craftable Vic. I'm talking about "hope" for killing sentient pandora. Also every decent miner would hunt maint for this now, since mood is more important to miner that anyone else.

**Comment by VictorVow** [comment]
This is a tiny fraction of what Hope provides in terms of mood boost, otherwise every crew member would game it I know.

**Comment by Isy232** [comment]
> These are nice, but I question why we weren't just throwing out players for validing off such unreliable information?

The issue is if you see X go and grab a bunch of poppies, you know X likely is a heretic.

You're a good player, your not going to immediately attack them or attempt to valid hunt them.

But you still are pretty damn sure X is a heretic and subconciously might not want to sit around near them more because of it and might be more willing to call security over if they act a little suspcious.

This is a good thing.

**Comment by hotimep** [comment]
>These are nice, but I question why we weren't just throwing out players for validing off such unreliable information?

better to get rid of the problem from the source

**Review by francinum — DISMISSED** [review]

File: code/datums/mood_events/generic_positive_events.dm [hunk 1]
```diff
@@ -171,6 +171,10 @@
 /datum/mood_event/nanite_happiness/add_effects(message)
 	description = "<span class='nicegreen robot'>+++++++[message]+++++++</span>\n"
 
+/datum/mood_event/poppy_pin
+	description = "<span class='nicegreen'>I feel proud to show my remembrance of the many who have died to ensure that I have freedom.</span>\n"
+	mood_change = 0.3
```
Comment (PowerfulBacon) [inline code/datums/mood_events/generic_positive_events.dm]: I don't think 0.3 is a good idea, all over instances of mood_change are integers

Comment (VictorVow) [reply code/datums/mood_events/generic_positive_events.dm]: 1 is quite significant for such a simple item. What kind of downside is there from having it not be an integer here?

Comment (PowerfulBacon) [reply code/datums/mood_events/generic_positive_events.dm]: none probably, but 1 is literally nothing

**Comment by utilbird** [comment]
Will merge after bacon's review is addressed

File: code/datums/mood_events/generic_positive_events.dm (hunk 1 above)
Comment (VictorVow) [reply code/datums/mood_events/generic_positive_events.dm]: Alright, changed.

**Comment by VictorVow** [comment]
Made the requested changes.

**Review by utilbird — APPROVED** [review]

**Pull request merged**